src/internal/storage/fileset/unordered_writer.go

```diff
@@ -93,11 +93,11 @@ func (uw *UnorderedWriter) validate(p string) error {
 
 // serialize will be called whenever the in-memory file set is past the memory threshold.
 // A new in-memory file set will be created for the following operations.
-func (uw *UnorderedWriter) serialize() error {
+func (uw *UnorderedWriter) serialize(ctx context.Context) error {
 	if uw.buffer.Empty() {
 		return nil
 	}
-	return miscutil.LogStep("UnorderedWriter.serialize", func() error {
+	return miscutil.LogStep(ctx, "UnorderedWriter.serialize during %s", func() error {
```

Comment:
This function doesn't apply format args, so the raw logs end up having %s in them.  (This ends up being the `name` argument to `entry.Infof("started %v", name)` etc.)

I think you will need to fmt.Sprintf whatever you want %s to be here just like you do at line 200 below.

Reply:
Argh, that was a leftover mistake that I did not catch. Thanks for the good eye.